core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3564,19 +3581,28 @@ class KafkaApis(val requestChannel: RequestChannel,
 
     val responseData = new LiCombinedControlResponseData()
 
+    def updateMetricsOnDecomposedRequestComplete(apiKey: ApiKeys, requestDequeueTimeNanos: Long, apiLocalCompleteTimeNanos: Long): Unit = {
```

Comment:
requestDequeueTimeNanos  -> requestProcessingStartTimeNanos?
since it's really processing starttime, not dequeue time.

Reply:
Fixed